lldb/include/lldb/Utility/FileSpec.h

```diff
@@ -430,6 +436,8 @@ class FileSpec {
     No
   };
 
+  /// The download time of the file.
+  double m_download_time = 0.0;
```

Comment:
We store a bunch of FileSpec objects in LLDB, we don't want to increase the size of the FileSpec class by `sizeof(double)`. So we shouldn't store this in FileSpec.